io.openems.edge.evcs.api/src/io/openems/edge/evcs/v2/api/Priority.java

```diff
@@ -0,0 +1,47 @@
+package io.openems.edge.evcs.v2.api;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum Priority implements OptionsEnum {
+    /**
+     * priority is unknown.
+     */
+    UNDEFINED(-1, "Undefined"), //
+    /**
+     * EVCS is used for cars with long standing times.
+     */
+    LOW(1, "Low"), //
+    /**
+     * EVCS is used for cars with average standing times.
+     */
+    MEDIUM(2, "Medium"), //
+    /**
+     * EVCS should be charged as fast as possible.
+     */
+    HIGH(3, "High") //
+    ;
```

Comment:
I would change MEDIUM to REGULAR or STANDARD or DEFAULT. Since Medium is the default one / the one everyone uses when no priority is required it should be something more readable. Medium seems confusing and SGReady already use REGULAR as a default for consistency.
```suggestion
public enum Priority implements OptionsEnum {
    /**
     * priority is unknown.
     */
    UNDEFINED(-1, "Undefined"), //
    /**
     * EVCS is used for cars with long standing times.
     */
    LOW(1, "Low"), //
    /**
     * EVCS is used for cars with average standing times.
     */
    REGULAR(2, "Regular"), //
    /**
     * EVCS should be charged as fast as possible.
     */
    HIGH(3, "High") //
    ;
```